src/protocol/definitions/ping360.json

```diff
@@ -110,65 +100,46 @@
                     }
                 ]
             },
-            "parameters": {
-                "id": "2302",
-                "description": "The Parameters message is sent from the sonar to the host in response to a request from the host.",
+            "ping360_data": {
```

Comment:
Since this messages are sensor specific, I believe that ping360 prefix here is not necessary.

Reply:
Do you want to suggest a different name. Plain `data` is not good.

Reply:
`profile` ? At least looks like a ping1D profile message.

Reply:
how about `device_data` or `sonar_data`. It will not always contain a profile (zero data length) it serves also the purpose of communicating only the current settings/parameter block.

This message is used for access of all the device data except firmware which is why I named it ping360_data in the first place.